joomla/joomla-cms
# Redirect plugin

### Summary of Changes
This PR adds the functionality to exclude specific urls in the redirect extension. For example I got a lot of wordpress hacking attemps on my Joomla! page (wp-admin) which are polluting my database. Now it's possible to exclude this things

### Testing Instructions
1. Apply this patch
2. Activate the redirect plugin
3. In the plugin you can either define a term or an regular expression (check the checkbox) for the string to seach
4. Call a page which does not exists

Examples:
normal term: abc
regular expression: [x]{2}


### Expected result
your-domain.tld/abc => will not be saved in the database
your-domain.tld/xx => will not be saved in the database

your-domain.tld/x => will be saved in the database
your-domain.tld/yy => will be saved in the database
